### PR TITLE
[13.0][MIG] sale_order_import_ubl

### PR DESCRIPTION
**Pull request by sebalix**
Depends on:
- [x] https://github.com/OCA/edi/pull/201

I have to change the way the ZIP codes are formatted in the demo data, otherwise partners didn't match... This is caused "because" of this line: https://github.com/OCA/edi/blob/13.0/base_ubl/models/ubl.py#L659
but I do not understand why tests were not failing on previous versions as this code is there from at least 8.0... any idea?

### CODE REVIEW

File: sale_order_import_ubl/wizard/sale_order_import.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    @api.model
+    def parse_xml_order(self, xml_root, detect_doc_type=False):
+        start_tag = "{urn:oasis:names:specification:ubl:schema:xsd:"
+        rfq = "RequestForQuotation"
+        if xml_root.tag == start_tag + "Order-2}Order":
+            if detect_doc_type:
+                return "order"
+            else:
+                return self.parse_ubl_sale_order(xml_root)
+        elif xml_root.tag == "{}{}-2}}{}".format(start_tag, rfq, rfq):
```
Comment (i-vyshnevska) [inline sale_order_import_ubl/wizard/sale_order_import.py:21]: what does this "2" for?

**Review by Tonow-c2c — APPROVED** [review]

**Review by jgrandguillaume — APPROVED** [review]
LGMT

File: sale_order_import_ubl/wizard/sale_order_import.py (hunk 1 above)
Comment (sebalix) [reply sale_order_import_ubl/wizard/sale_order_import.py:21]: I don't know, it respects the UBL specs I suppose, I didn't change that.

**Review by desdelinux — APPROVED** [review]
functional review is ok

**Pull request merged**